include/pagmo/population.hpp

```diff
@@ -287,6 +288,24 @@ class population
         return worst_idx(tol_vector);
     }
 
+    /// Gets the champion decision vector
+    vector_double get_champion_x() const {
+        if (m_prob.get_nobj() > 1u) {
+            pagmo_throw(std::invalid_argument,
+                        "The Champion of a population can only be extracted in single objective problems");
+        }
+        return m_champion_x;
```

Comment:
I do not think it matters much. EIther solutions will do, maybe I prefer the returning empty vector as it offers more coding "flexibility"

has_champion() seems like an overkill ....